wgengine/magicsock/magicsock_test.go

```diff
@@ -1329,11 +1323,11 @@ func TestSetNetworkMapChangingNodeKey(t *testing.T) {
 	var buf tstest.MemLogger
 	conn.logf = buf.Logf
 
-	conn.SetPrivateKey(wgkey.Private{0: 1})
+	conn.SetPrivateKey(key.NewNode())
 
 	discoKey := tailcfg.DiscoKey{31: 1}
-	nodeKey1 := tailcfg.NodeKey{0: 'N', 1: 'K', 2: '1'}
-	nodeKey2 := tailcfg.NodeKey{0: 'N', 1: 'K', 2: '2'}
+	nodeKey1, _ := key.ParseNodePublicUntyped(mem.S("1000000000000000000000000000000000000000000000000000000000000000"))
```

Comment:
This is the second or third time I've seen something like this. I wonder whether we should have an API to get a "memorable"/"readable"'/"numbered" key. The API could include the word "Test" in it. Or even require you to pass in a testing.TB, which is hard to get ahold of when not in a test.

Reply:
Will do separate from the big refactor PR that has constant rebase conflicts: https://github.com/tailscale/tailscale/issues/2889